twitter_like_app/src/pages/Login/Login.jsx

```diff
@@ -1,23 +1,56 @@
-import React,{Component} from "react";
-import {Grid,Paper,Avatar,Button} from '@material-ui/core';
+import React, { Component } from "react";
+import './Login.css';
+import { Grid, Paper, Avatar, TextField, Button } from '@material-ui/core';
 import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import Checkbox from '@mui/material/Checkbox';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Link from '@mui/material/Link';
 import Typography from '@mui/material/Typography';
-import './Login.css';
+import { Formik, Field, Form, ErrorMessage } from 'formik';
+import * as Yup from 'yup';
+
 
 class Login extends Component {
-  
   render() {
+    const validationSchema = Yup.object().shape({
+      username: Yup.string().email('please enter valid email').required('Required'),
+      password: Yup.string().required("Required")
+    });
+    const onSubmit = (value, props) => {
+      setTimeout(() => {
+        props.resetForm()
+        props.setSubmitting(false)
+      }, 2000)
+
+    }
+    const initialValues = {
+      username: '',
+      password: '',
+      remember: 'false'
```

Comment:
Move initial values above the component. There is no need to reinitialize it every time the component rerenders